ktxspec.adoc

```diff
@@ -1288,8 +1296,8 @@ in the <<_level_index,Level Index>>.
 The offset of the start of the alpha slice within the
 <<_levelimages,levelImages>> of its mip level and its byte length.
 
-If `hasAlphaSlices` is unset these values must be 0. If `isETC1s` and
-`hasAlphaSlices` are set these values must be non-zero.
+If the data format descriptor's colorModel is `KHR_DF_MODEL_ETC1S`
+and it has only one sample these values must be 0.
```

Comment:
Sample or plane?

Reply:
I've updated this to

```
If there is only one slice these values must be 0. For ETC1S with the data
format descriptor colorModel of `KHR_DF_MODEL_ETC1S` this corresponds to
the DFD having only one sample. (As the format is supercompressed
`bytesPlane` fields can't be used to determine the number of planes.)
```

I also added a tip in the ETC1S section.